src/tir/schedule/primitive/compute_at.cc

```diff
@@ -514,6 +605,14 @@ void ComputeAtOrReverseComputeAtImpl(ScheduleState self, const StmtSRef& block_s
       /*realize=*/reconstructor.new_block_realize_,
       /*loop_var_ranges=*/LoopDomainOfSRefTreePath(GetRef<StmtSRef>(block_sref->parent)),
       /*analyzer=*/&analyzer);
+  // The verifier can not prove region cover state if some complex predicte is introduced
+  // so here it explicitly reset these flags below.
+  if (is_compute_at && !is_const_int(reconstructor.new_block_realize_->predicate)) {
```

Comment:
It's a bug of `RegionCoverCheck`. We should fix it instead of working around it.